src/main/java/com/rarchives/ripme/ui/MainWindow.java

```diff
@@ -247,7 +247,10 @@ public void run() {
     }
 
     private void createUI(Container pane) {
-        setupTrayIcon();
+        //If creating the tray icon fails, ignore it.
+        try {
+            setupTrayIcon();
+        } catch (Exception e) { }
```

Comment:
Yay pokemon exceptions ("Gotta `catch` em all!"). Ugly but effective! :P

Reply:
Pokemon didn't even come into mind when I coded this.

Totally agree it's ugly but is there any other way ? I were thinking it's better to abort the whole thing no matter what happens cuz it means the icon initialization didn't go flawlessly => the icon may not even work.

Reply:
Yeah, I think ignoring the exception in this case is the proper way to handle an exception here.